src/react/account/AccountList.jsx

```diff
@@ -65,7 +65,7 @@ function AccountList({ accountList, accountIndex }: Props) {
         if (listRef && listRef.current && accountIndex > -1) {
             listRef.current.scrollToItem(accountIndex, 'smart');
         }
-    }, [listRef]);
+    }, [listRef, accountIndex]);
```

Comment:
`accountIndex` was not mentioned on purpose. 
We want to scroll to the selected item only once on mount. 
We can add a comment. :)